Makefile

```diff
@@ -164,7 +164,8 @@ update_databases: update_amrfinderplus \
 	update_mlst_db \
 	update_blast_db \
 	update_finder_dbs \
-	update_shigapass_db
+	update_shigapass_db# \
+	# update_hostile_db
```

Comment:
shouldnt the Makefile update the hostile database? If not please remove the comment

Reply:
It doesn't work in the singularity image at the moment. I have contacted the developers to tell them that there's a problem.